### PR TITLE
IBX-2559: Added new style to ez-extra-actions__content class to allow scrolling language list

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question      | Answer
| ------------- | ---
| Tickets       | [IBX-2559](https://issues.ibexa.co/browse/IBX-2559)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Added new class to scrolling language lists

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/themes/admin/content/widget/content_edit.html.twig [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <div class="ez-extra-actions ez-extra-actions--edit ez-extra-actions--hidden {% if form.language.vars.choices|length == 1 %} ez-extra-actions--prevent-show {% endif %}" data-actions="edit">
     <div class="ez-extra-actions__header">{{ 'content.edit.select_language'|trans|desc('Select translation') }} ({{ form.language.vars.choices|length }})</div>
-    <div class="ez-extra-actions__content">
+    <div class="ez-extra-actions__content ez-extra-actions__content__scroller">
```
Comment (OstafinL) [inline src/bundle/Resources/views/themes/admin/content/widget/content_edit.html.twig]: Why not just?
```suggestion
    <div class="ez-extra-actions__content">
```
You have selector in https://github.com/ezsystems/ezplatform-admin-ui/blob/b048b14e7ecc8af1d62813c47aa4b63f2f4eb579/src/bundle/Resources/public/scss/_extra-actions.scss#L54

Comment (mateuszdebinski) [reply src/bundle/Resources/views/themes/admin/content/widget/content_edit.html.twig]: there is more than one place where this class is used, so I didn't want to change it everywhere. However, if there is no problem with it, I will move code to __content selector

Comment (dew326) [reply src/bundle/Resources/views/themes/admin/content/widget/content_edit.html.twig]: BEM element cannot be parent and child at the same time. In that case, you should use a modifier.
`ez-extra-actions__content--scroller`

edit: maybe in other places there is the same problem and it should be done globally on __content as Łukasz suggested? But this is something to check.

**Review by GrabowskiM — APPROVED** [review]
Now PR title isn't correct, as you don't add any new class

**Review by OstafinL — APPROVED** [review]

**Review by bogusez — APPROVED** [review]

**Pull request merged**